src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -797,26 +807,22 @@ export default class CanvasController {
         cc.save();
 
         if (!aircraft.inside_ctr) {
-            cc.fillStyle = this.theme.RADAR_TARGET_OUTSIDE_RANGE;
+            cc.fillStyle = this.theme.RADAR_TARGET.HISTORY_DOT_OUTSIDE_RANGE;
         } else {
-            cc.fillStyle = this.theme.RADAR_TARGET_IN_RANGE;
+            cc.fillStyle = this.theme.RADAR_TARGET.HISTORY_DOT_INSIDE_RANGE;
         }
-
         const length = aircraft.relativePositionHistory.length;
         for (let i = 0; i < length; i++) {
```

Comment:
we should re-assign `aircraft.relativePositionHistory[i]` to some other contextual var here (like `fix` in the other `for` loop above).  We should also give the `0` and `1` indexes an enumerated name, like `x` and `y`.  Otherwise it's a little hard to tell what those actually are.